### DISCUSSION TITLE
For the Record/SB/CT results?

### DISCUSSION THREAD

**Discussion by profmadden** [The-OpenROAD-Project/OpenROAD#2845] [General]
Hi -- I'm looking through a draft for the upcoming ISPD "Assessment of Reinforcement Learning" paper.

If there are placement results for the ICCAD04 testcases handy, I'd appreciate a link to them.  I'd love to see the actual placements from CT, SA, and RePLAce, but don't have the time and resources (and expertise) to run all of the experiments myself.  Bookshelf format is preferred, but I could probably figure out a way to convert from ProtoBuf if needed.  There seem to have been some slight tweaks to the macro block sizes over the years -- so input files would be helpful too.

Thanks to everyone for all the work on this.  I've done a lot of benchmarking over the years, and know that it's generally a thankless task -- so seriously, thanks very much for all the work.  I'm sure it's zero fun and a huge time sink.

**Comment by maliberty** [comment]
I'm not aware of such results being handy unless @mgwoo has some.
